apps/arweave/include/ar_pool.hrl

```diff
@@ -0,0 +1,35 @@
+%% The number of VDF steps ("jobs") the pool server serves at a time.
+-define(GET_JOBS_COUNT, 10).
+
+%% The time in seconds the pool server waits before giving up on replying with
+%% new jobs when the client already has the newest job.
+-define(GET_JOBS_TIMEOUT_S, 2).
+
+%% The frequency in milliseconds of asking the pool proxy or CM exit node about new jobs.
+-define(FETCH_JOBS_FREQUENCY_MS, 5000).
+
+%% The time in milliseconds we wait before retrying a failed fetch jobs request.
+-define(FETCH_JOBS_RETRY_MS, 2000).
+
+%% @doc A collection of mining jobs.
+-record(jobs, {
+	vdf = [], % The information about a single VDF output (a "job").
```

Comment:
Can we name this `job` instead of `vdf`?

Reply:
Hm, actually it looks like we refer to this as 3 diferent things: `vdf`, `job`, and then later `triplet`. Can we unify the 3 different names?

What about moving the `get_step_triplets` and related functions out of `ar_nonce_limiter` and putting them in `ar_pool` and then using the term `job` everywhere (i.e. replace `vdf` here, and `triplet` there with `job`)

Reply:
Okay now having read though `ar_pool.erl` I'm not sure of naming. But I do think we can clarify things... triplet vs. vdf vs. job vs. the 5-tuple in ar_pool.erl (https://github.com/ArweaveTeam/arweave/pull/515/files#r1444973868) all feel pretty similar to me.


What about using the `job` record here to store the 5 values from the 5-tuple in `ar_pool.erl`... and then maybe coming up with a better name for `triplet`? I kinda feel like the full definition of a "step" actually includes `output`, `step_number`, `partition_upper_bound` and `seed`... 

Not sure... but I am having trouble keeping all the different terms straight and when they overlap vs. differ... so I"m expecting some pain down the road once we go live and have to start fixing bugs